website/src/components/EventSummary/EventSummary.test.jsx

```diff
@@ -20,3 +21,17 @@ describe('EventSummary', () => {
     expect(renderer.create(<EventSummary event={event} />).toJSON()).toMatchSnapshot();
   });
 });
+
+/* Enzyme */
+const wrapper = shallow(<EventSummary />);
+describe('EventSummary shallow', () => {
+  it('Course should exist', () => {
+    expect(wrapper.find('.course').closest('.EventSummary').length).toBe(1);
+  });
+  it('Eventname should exist', () => {
```

Comment:
These tests are only checking the class exists but not the actual item. You should modify the test so that it checks for the actual event name instead of the eventname class

Reply:
Mainly, as we discussed before, all the className and position is tested by the snapshot, therefore, the Enzyme Test is towards to checking some button, text, props, and so on.